test/checkout.int.t.sol

```diff
@@ -1119,8 +1171,150 @@ contract Checkout_swapFromSupplierWithoutFeeSpecialOrder is Checkout {
     assertEq(removalAmounts[0], certificateAmount);
     assertEq(
       _bpNori.balanceOf(_namedAccounts.supplier),
-      (certificateAmount * customPriceMultiple) / 100 / 2 // divide to account for price multiple scale and then holdback percentage of 50%
+      (certificateAmount * customPriceMultiple) / 100 / 2 // divide to account for price multiple scale and holdback percentage of 50%
```

Comment:
Nit: use variable names to self document (e.g., holdbackPercent, priceMultipleScale)